server/punishments.js

```diff
@@ -67,10 +67,12 @@ class PunishmentMap extends Map {
 	 * @param {(punishment: Punishment, id: string, map: PunishmentMap) => void} callback
 	 */
 	forEach(callback) {
-		super.forEach((punishment, k) => {
-			if (Date.now() < punishment[2]) return callback(punishment, k, this);
+		for (const [k, punishment] of super.entries()) {
+			if (Date.now() < punishment[2]) {
+				return callback(punishment, k, this);
```

Comment:
Why did you change this back? It's supposed to be `callback` and then `continue`

Reply:
Because, it's an eslint build error
https://eslint.org/docs/rules/callback-return

Reply:
Just `eslint-disable` it.

Don't write bad code just because ESLint tells you to. :|